src/atoms/Text/Text.web.js

```diff
@@ -1,3 +1,140 @@
-const Text = () => null;
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import baseTheme from '../../tokens/theme';
+import { getColorKeys, getColor, getLineHeight } from '../../_helpers/theme';
+
+/*
+ * Sets height for the div
+ * @param {Object} theme
+ * @param {String} _lineHeight
+ * @param {number} maxLines
+ */
+const calculateHeight = ({ theme, _lineHeight, maxLines }) => {
+  const lineHeight = `${parseFloat(theme.fonts.lineHeight[_lineHeight])}`;
+  const containerHeight = `${lineHeight * maxLines}px`;
+
+  return containerHeight;
+};
+
+const styles = {
+  fontFamily({ theme, _weight }) {
+    return theme.fonts.family.lato[_weight];
+  },
+  fontSize({ theme, size }) {
+    return theme.fonts.size[size];
+  },
+  color({ theme, color }) {
+    return getColor(theme, color);
+  },
+  align({ align }) {
+    switch (align) {
+      case 'left':
+      default:
+        return 'flex-start';
+      case 'center':
+        return 'center';
+      case 'right':
+        return 'flex-end';
+    }
+  },
+  letterSpacing({ theme, _letterSpacing }) {
+    return `${theme.fonts.letterSpacing[_letterSpacing]}`;
+  },
+  lineHeight({ theme, size, _lineHeight }) {
+    if (_lineHeight) {
+      return theme.fonts.lineHeight[_lineHeight];
+    } else {
+      return getLineHeight(theme, size);
+    }
+  },
+  maxHeight({ theme, _lineHeight, maxLines }) {
+    if (maxLines) {
+      const maxHeight = calculateHeight({ theme, _lineHeight, maxLines });
```

Comment:
This lineHeight is not correct. use ```getLineHeight(theme, size) ``` because lineHeight is based on the size. The prop _lineHeight is used only in exceptional cases and shouldn't be used anywhere at all otherwise

Reply:
done

Reply:
Why did we create a new function here? The rule of thumb is create functions only if you write the same logic more than 2 times and over here we are just using it at once place that too in the same component. This affects readability.